src/Main/Views/Income/ChartD3js/ChartD3.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from 'react';
+import * as d3 from "d3";
+
+import chartData from "../data/chartData";
+
+import "./ChartD3.css";
+
+
+
+const ChartD3 = () => {
+
+    let data = Object.values(Object.entries(chartData)[0][1]);
+    data = data.map(el => { return Math.round(el) });
+    let labels = []
+    labels.length = 26;
+    labels.fill(null);
+    labels = labels.map((el, idx) => { return el = idx });
```

Comment:
W zasadzie to nie musisz wykonywać tego el = idx. Domyślnie to co jest za return zastępuje elementy :) Całość w zasadzie mogłabyś skrócić do:
```labels.map((el, idx) => idx);```

Reply:
- [x] Done